add missing sys/sysmacros.h for major, minor and makedev functions

diff --git a/src/daemon.c b/src/daemon.c
index fafcf9a..bbdce2e 100644
--- src/daemon.c
+++ src/daemon.c
@@ -48,6 +48,7 @@
 #include <net/if_arp.h>
 #include <fcntl.h>
 #include <signal.h>
+#include <sys/sysmacros.h>
 
 #include <glib.h>
 #include <glib/gi18n-lib.h>
diff --git a/src/device.c b/src/device.c
index 2ae7f38..33c8193 100644
--- src/device.c
+++ src/device.c
@@ -41,6 +41,7 @@
 #include <sys/ioctl.h>
 #include <linux/cdrom.h>
 #include <linux/loop.h>
+#include <sys/sysmacros.h>
 
 #include <glib.h>
 #include <glib/gstdio.h>
diff --git a/src/mount-monitor.c b/src/mount-monitor.c
index 573a69c..0a0e68e 100644
--- src/mount-monitor.c
+++ src/mount-monitor.c
@@ -30,6 +30,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <mntent.h>
+#include <sys/sysmacros.h>
 
 #include <glib.h>
 #include <glib/gi18n-lib.h>
diff --git a/src/probers/part-id.c b/src/probers/part-id.c
index 658e7d9..ecebbff 100644
--- src/probers/part-id.c
+++ src/probers/part-id.c
@@ -27,6 +27,7 @@
 #include <sys/wait.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <sys/sysmacros.h>
 
 #include <libudev.h>
 
diff --git a/tools/umount-udisks.c b/tools/umount-udisks.c
index 2813fe0..11a8fff 100644
--- tools/umount-udisks.c
+++ tools/umount-udisks.c
@@ -33,6 +33,7 @@
 #include <fcntl.h>
 #include <pwd.h>
 #include <grp.h>
+#include <sys/sysmacros.h>
 
 #include <glib.h>
 #include <glib/gi18n-lib.h>

diff --git a/tools/udisks.c b/tools/udisks.c
index 97e80d7..570d5e2 100644
--- tools/udisks.c
+++ tools/udisks.c
@@ -35,6 +35,7 @@
 #include <pwd.h>
 #include <grp.h>
 #include <locale.h>
+#include <sys/sysmacros.h>
 
 #include <glib.h>
 #include <glib/gi18n-lib.h>

